yarn.lock

```diff
@@ -1786,8 +1786,8 @@
     react-transition-group "^4.4.5"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.14.2"
-  resolved "https://github.com/mui/material-ui.git#5bac8b3c66ae144c35609129e8391c1653bf0aae"
+  version "5.14.3"
+  resolved "https://github.com/mui/material-ui.git#1d1e96d1bc2542cdfd1cc9fde415fc11d682e966"
```

Comment:
Should we remove the icons?
https://github.com/mui/mui-x/blob/b27df74db171a2af42aa087a61634e1ca57b0a01/docs/data/pages.ts#L21

<img width="299" alt="Screenshot 2023-08-02 at 14 16 31" src="https://github.com/mui/mui-x/assets/13808724/bceef46c-d182-4a9a-9f4c-fac1c1f3bd09">

Reply:
🙈